company_website/static/common/scripts/set_current_year_in_footer.js

```diff
@@ -0,0 +1,3 @@
+let current_year = new Date().getUTCFullYear().toString()
```

Comment:
In javascript convention about naming is camelCase instead of snake_case ;) Please fix it

Reply:
Done